### PR TITLE
Add acquirer merchant documentation

### PR DESCRIPTION
**Pull request by Magnus93**

### CODE REVIEW

**Review by mayask — APPROVED** [review]

**Comment by mayask** [comment]
@euqen how do you think merchant docs should sit under Integrate or should be separate ?

**Comment by euqen** [comment]
@mayask I think it should be separate for now. The persona who integrates against Merchant Api is our direct customer. But if we talk about acquiring API that we provide to 3rd party PSPs - this is what we could consider providing in integrate

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ module.exports = {
       '/integrate/onboarding/': getIntegrateSidebar(),
       '/integrate/card-programmes/': getIntegrateSidebar(),
       '/integrate/embedded-finance/': getIntegrateSidebar(),
-      '/integrate/merchants/': getIntegrateSidebar(),
+      '/integrate/merchants-reference/': getIntegrateSidebar(),
```
Comment (euqen) [inline docs/.vuepress/config.js:59]: please remove integrate prefix here

Comment (euqen) [reply docs/.vuepress/config.js:59]: and below

File: docs/integrate/merchants-reference/authorization.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Authorization Creatable
+
+| Property     | Type                                                      | Description                   |
+|--------------|-----------------------------------------------------------|-------------------------------|
+| `number`     | `string`                                                  | has to be a unique identifier |
+| `amount`     | `number`                                                  |                               |
+| `currency`   | [`Currency`](./other.html#currency)                       |                               |
+| `card`       | `Signed JWT` or [`Card.Creatable`](./card.html#creatable) |                               |
+| `descriptor` | `string`                                                  | (optional)                    |
+| `recurring`  | [`Recurring`](#recurring)                                 | (optional)                    |
```
Comment (euqen) [inline docs/integrate/merchants-reference/authorization.md:15]: optional or required must be separate column.

File: docs/integrate/merchants-reference/authorization.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+| `void`       | [`DateTime`](./other.html#datetime)                              | (optional)       |
+| `status`     | [`Partial<Record<Status, number>>`](./authorization.html#status) |                  |
+
+
+### Status
+Authorization.Status is string set to `"authorized"`, `"cancelled"`, `"captured"`, `"refunded"` or `"settled"`.
+
+### Recurring
+Recurring can be defined in four ways: 
+ - As a string: `"initial"`
```
Comment (euqen) [inline docs/integrate/merchants-reference/authorization.md:44]: please describe which fields will have such values

File: docs/.vuepress/config.js (hunk 1 above)
Comment (euqen) [reply docs/.vuepress/config.js:59]: So it should be as a separate section in docs that does not belong to integrate

Comment (Magnus93) [reply docs/.vuepress/config.js:59]: Ok, I will add seperate folders:
`/merchant/reference/`
`/merchant/api/`

**Pull request merged**